src/dumpty/pipeline.py

```diff
@@ -326,21 +328,41 @@ def introspect(self, extract: Extract) -> Extract:
             if is_numeric and full_introspect:
                 logger.debug(
                     f"Getting min({pk.name}), max({pk.name}), and count(*) of {extract.name}")
-                qry = session.query(func.max(pk).label("max"),
-                                    func.min(pk).label("min"),
-                                    count_fn(
-                                        literal_column("*")).label("count")
-                                    ).select_from(table)
-                res = qry.one()
-                extract.max = res.max
-                extract.min = res.min
-                extract.rows = res.count
+                
+                if self.config.fastcount:
+                    qry = session.query(func.max(pk).label("max"),
+                                        func.min(pk).label("min")
+                                        ).select_from(table)
+                    res = qry.one()
+                    extract.max = res.max
+                    extract.min = res.min
+                    
+                    result = session.execute(f"EXEC sp_spaceused N'dbo.{extract.name}';").fetchall()
```

Comment:
Since this is a full introspection, and it has to find min/max anyway, don't use fastcount here.